lib/split.rb

```diff
@@ -66,4 +66,11 @@ def configure
   end
 end
 
-Split.configure {}
+# Check to see if being run in a Rails application.  If so, wait until before_initialize to run configuration so Gems that create ENV variables have the chance to initialize first.
+if defined?(Rails::Railtie)
```

Comment:
Can you change this to check for `Rails` instead?

```ruby
if defined?(::Rails)
```

See other gems for example: https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/rails.rb#L51 and https://github.com/brandonhilkert/sucker_punch/blob/master/lib/sucker_punch.rb#L57

Reply:
Sure thing!  I thought being explicit (since we will be inheriting from Railties) was better, but seeing the convention of checking for ::Rails seems like a good idea.  Updated!